.travis.yml

```diff
@@ -6,11 +6,15 @@ matrix:
     - php: 7.2
     - php: 7.3
     - php: 7.4
-    - php: 8.0snapshot
+    - php: 8.0
     - php: 7.1
       env: SYMFONY_VERSION='3.4.*'
     - php: 7.1
       env: DEPENDENCIES=beta
+    - php: 8.0
+      env:
+        - DEPENDENCIES=dev
+        - SYMFONY_VERSION='6.0.x-dev'
```

Comment:
```suggestion
        - SYMFONY_VERSION='6.0.*'
```

?

Reply:
Yep, now I can change to `*` since Symfony 6 has been released :+1:

Reply:
I've re-added `6.0.x-dev` until Symfony 6.0.1 is released.